### PR TITLE
[MP v2] Migrate Search app's artworks tab

### PR DESCRIPTION
**Pull request by alloy**
Part of https://artsyproduct.atlassian.net/browse/PLATFORM-1960
Relies on https://github.com/artsy/metaphysics/pull/2042

### CODE REVIEW

File: src/Apps/Search/SearchApp.tsx [hunk 1]
```diff
@@ -60,9 +62,7 @@ export class SearchApp extends React.Component<Props> {
     return (
       <>
         <Row>
-          <Col>
-            <RecentlyViewed />
-          </Col>
+          <Col>{/*<RecentlyViewed />*/}</Col>
```
Comment (alloy) [inline src/Apps/Search/SearchApp.tsx]: TODO: This was just temporarily disabled in order to focus on getting the first search page to render.

**Comment by mzikherman** [comment]
Nice! Did the filter component wind up working as-is (pagination and filtering)?

**Comment by alloy** [comment]
Errrr, no this is very much a WIP, but I forgot to label it as such 😅 Thanks for the poke!

**Comment by mzikherman** [comment]
Was about to come back and see if we should merge anyway, and continue iterating (several things in the filter probably need to be updated and that affects several apps).

I was about to merge and move on from the artist page since the artist filter is a special case of the general filter component (aggregations stuff), and was going to leave that for the end.

File: src/Apps/Search/Routes/Artworks/index.tsx [hunk 2]
```diff
@@ -1,28 +1,25 @@
...
-interface SearchResultsRouteProps {
+interface SearchResultsRouteProps extends RouterState {
   viewer: ArtworkFilter_viewer
-  location: Location
 }
 
-export const SearchResultsArtworksRoute: React.FC<
-  SearchResultsRouteProps
-> = props => {
+export const SearchResultsArtworksRoute = withRouter((props => {
```
Comment (mzikherman) [inline src/Apps/Search/Routes/Artworks/index.tsx]: 👍

File: src/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx [hunk 3]
```diff
@@ -240,55 +243,58 @@ export const ArtworkFilterContextProvider: React.FC<
...
+          majorPeriods: value ? [value] : [],
         }
       }
       if (name === "page") {
         filterState[name] = Number(value)
       }
 
       // String filter types
-      ;[
+      const stringFilterTypes: Array<keyof ArtworkFilters> = [
```
Comment (damassi) [inline src/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx]: @alloy - thanks for updating with more type coverage 👍

Comment (alloy) [reply src/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx]: My pleasure 👍

**Comment by artsyit** [comment]
:rocket: PR was released in v22.0.0 :rocket:

**Pull request merged**